openreview/conference/matching.py

```diff
@@ -929,7 +933,50 @@ def setup(self, affinity_score_file=None, tpms_score_file=None, elmo_score_file=
         if build_conflicts:
             self._build_conflicts(submissions, user_profiles, openreview.tools.get_neurips_profile_info if build_conflicts == 'neurips' else openreview.tools.get_profile_info)
 
+        if scores_stream:
```

Comment:
this code is very similar to the code inside `_build_note_scores()`

Reply:
I would re-define two functions:

_build_note_scores_from_file
_build_note_scroes_from_stream

and define a function _build_note_scores(self, score_invitation_id, scores, submissions)

where scores is an array of scores than can be built from a file or from a stream:

```
scores = [input_line for input_line in scores_stream.decode().split('\n')]
```

or 

```
with open(score_file) as file_handle:
      scores = [row for row in csv.reader(file_handle)]
```